### DISCUSSION TITLE
Update jekyll-seo-tag to 2.6.x

### DISCUSSION THREAD

**Discussion by luciferous** [community/community#22990] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>It looks like Github Pages uses the plugin jekyll-seo-tag@2.5.0, and I’m wondering what the update schedule is for upgrading to a newer release, e.g., jekyll-seo-tag@2.6.x.</p>
<p>v2.6.x adds twitter:image, and twitter:card, among <a href="https://github.com/jekyll/jekyll-seo-tag/compare/v2.5.0..v2.6.1" rel="nofollow noopener">other nice things</a>.</p>
<p>Does anyone know where to find this out?</p>
<p>Thanks</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>The <a href="https://github.com/github/pages-gem" rel="nofollow noopener">github-pages gem</a> is maintained in a public repository. You can ask there, though there may be some reason that we can’t upgrade all GitHub Pages users to a newer version of some dependency.</p>
<p>Additionally, you don’t have to depend on the plugins that GitHub Pages provides. You can run a Jekyll build process locally using whatever plugins or other code you wish and deploy the output of that build to Pages yourself. It is a bit more complicated to do, but you don’t have to wait on us to use whatever plugins or plugin versions you choose.</p>
<p>I hope that helps!</p>

**Comment by luciferous** [comment]
<p>Thanks! I’ll open a pull request. I know I can build the site manually, but I’m trying to keep operational overhead at zero, so I’m good to stick with the defaults.</p>
